crates/holochain_sqlite/src/stats.rs

```diff
@@ -2,11 +2,19 @@ use crate::error::DatabaseError;
 use rusqlite::Transaction;
 
 pub fn get_size_on_disk(txn: Transaction) -> Result<usize, DatabaseError> {
-    txn.query_row("select sum(pgsize) from dbstat", (), |r| r.get(0))
-        .map_err(DatabaseError::SqliteError)
+    Ok(txn
+        .query_row("select sum(pgsize) from dbstat", (), |r| {
+            r.get::<_, Option<usize>>(0)
+        })
+        .map_err(DatabaseError::SqliteError)?
+        .unwrap_or_default())
```

Comment:
Is this not the same as using the `optional` method? see: https://docs.rs/rusqlite/latest/rusqlite/trait.OptionalExtension.html#tymethod.optional

```suggestion
    Ok(txn
        .query_row("select sum(pgsize) from dbstat", (), |r| r.get(0))
        .optional()
        .map_err(DatabaseError::SqliteError)?
        .unwrap_or_default())
```

Reply:
Nice, yes it is